textattack/loggers/attack_log_manager.py

```diff
@@ -19,15 +19,16 @@ def __init__(self):
 
     def enable_stdout(self):
         self.loggers.append(FileLogger(stdout=True))
+        return None
```

Comment:
Why do we return `None`?

Reply:
@cogeid please respond..

Reply:
yeah i'm not sure why that is there, will be fixed in the latest commit